hknweb/tours/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+from django.utils import timezone
+from django.conf import settings
+import datetime
+from django.core.validators import RegexValidator
+
+MAX_STRLEN = 255 
+MAX_TXTLEN = 2000 
+
+class DepTour(models.Model):
+    """
+    Model for a department tour
+    """
+
+    class Meta:
+        verbose_name = "Department Tour"
+
+    name            = models.CharField(max_length=MAX_STRLEN, default='')
+    email           = models.EmailField(max_length=MAX_STRLEN, default='')
+    confirm_email   = models.EmailField(max_length=MAX_STRLEN, default='')
+    desired_time    = models.TimeField(verbose_name='Desired Time')
+    date            = models.DateField(default=datetime.date.today,verbose_name='Desired Date')
+    phone_regex     = RegexValidator(regex=r'^\+?1?\d{10,15}$', message="Enter a valid 10-digit phone number in the format:'+999999999'")
```

Comment:
(same comment as above about the phone number)

It also looks like the regex supports a 10-15 digit phone number, which is totally fine, but it doesn't match the description which says only a 10 digit number